tests/vip-helpers/test-vip-elasticsearch.php

```diff
@@ -53,4 +53,34 @@ public function test__vip_elasticsearch_filter_ep_index_name__constant_is_false(
 
 		$this->assertEquals( 'index-name', $index_name );
 	}
+
+	/**
+	 * Test that we set a default bulk index chunk size limit
+	 *
+	 * @runInSeparateProcess
+	 * @preserveGlobalState disabled
+	 */
+	public function test__vip_elasticsearch_bulk_chunk_size_default() {
+		define( 'USE_VIP_ELASTICSEARCH', true );
```

Comment:
Should we add a test for when the constant is `false`? But I guess it doesn't really affect much even if the chunk size is set in that case

Reply:
Good idea - I've added that test in 1e3e6f22